projects/ngneat/hotkeys/src/lib/hotkeys-help/hotkeys-help.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+import { HotkeysService } from '../hotkeys.service';
+
+@Component({
+  templateUrl: './hotkeys-help.component.html',
+  styleUrls: ['./hotkeys-help.component.css']
+})
+export class HotkeysHelpComponent {
+  hotkeys: { group: string; keys: { desc: string; keys: string }[] }[] = [];
+
+  constructor(private hotkeysService: HotkeysService) {
+    const groups = this.hotkeysService.getShortcutsGroups();
```

Comment:
We should make the users' life easier, and expose a `getHotKeys()` method that returns the following structure:

```ts
[{
  group: name,
  keys: [
    { key, description }
  ]
}]
```